src/App.js

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import './App.css';
 import Calculator from './components/calculator';
+import Header from './components/Header';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
 
-// eslint-disable-next-line react/prefer-stateless-function
-class App extends React.Component {
+class App extends React.PureComponent {
   render() {
     return (
-      <Calculator />
+      <>
+        <Router>
+          <Header />
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="/Header" element={<Header />} />
+            <Route path="/quote" element={<Quotes />} />
+          </Routes>
+        </Router>
+        <Calculator />
```

Comment:
- [ ] Kindly remove the rendering of the ```calculator``` component from your ```App.js``` file so that the calculator does not display on your home page but shows only when the calculator link is clicked as the previous reviewer suggested.

### The project requirement:
<img width="471" alt="Screenshot 2022-08-19 at 03 08 58" src="https://user-images.githubusercontent.com/64774936/185527362-99ec7f46-4c67-4976-8c53-f6c56e1c3385.png">

### Your app:
<img width="1430" alt="Screenshot 2022-08-19 at 03 04 18" src="https://user-images.githubusercontent.com/64774936/185527491-bc82480f-c15a-4f55-b133-6dda0000e71d.png">

Reply:
i made the change but forgot to push it lol my bad